docs/lang_ref.rst

```diff
@@ -313,6 +313,33 @@ Maps are expressed with curly braces::
         example default
             similar_colors = {"blue": ["aqua", "azure"], "red": ["crimson"], "green": []}
 
+Map examples can also be multiline (Be mindful of indentation rules)::
+
+    struct SimpleDigits
+        digit_mapping Map(String, Int32)
+
+        example default
+            digit_mapping = # Mapping starts on line below
+                {
```

Comment:
It doesn't look very good to start a new line with a `LBRACE` (with or without indentation). Since there is no other place using `LBRACE`, maybe we can put some restriction here so that `LBRACE` always stick with the last line?

Reply:
good point, let me patch and fix w/ a new PR.

Reply:
After some thought, this is more like a coding style issue. It's fine to allow both styles at syntax level, like most other programming languages. But in the doc we should probably stick to a consistent coding style.

Reply:
can do.